upcloud/resource_upcloud_objectstorage_test.go

```diff
@@ -0,0 +1,231 @@
+package upcloud
+
+import (
+	"fmt"
+	"github.com/UpCloudLtd/upcloud-go-api/upcloud/request"
+	"github.com/hashicorp/go-retryablehttp"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/credentials"
+	"net/url"
+	"os"
+	"testing"
+	"time"
+)
+
+const expectedDescription = "My object storage"
+const expectedZone = "fi-hel2"
+const expectedKey = "an access key"
+const expectedSecret = "a secret key"
+
+const expectedName1 = "name1"
+const expectedName2 = "name2"
+
+
+func init() {
+	resource.AddTestSweepers("object_storage_cleanup", &resource.Sweeper{
+		Name: "object_storage_cleanup",
+		F: func(region string) error {
+			var nameMap = map[string]interface{} {
+				expectedName1: nil,
+				expectedName2: nil,
+			}
+
+			username, ok := os.LookupEnv("UPCLOUD_USERNAME")
+			if !ok {
+				return fmt.Errorf("UPCLOUD_USERNAME must be set for acceptance tests")
+			}
+
+			password, ok := os.LookupEnv("UPCLOUD_PASSWORD")
+			if !ok {
+				return fmt.Errorf("UPCLOUD_PASSWORD must be set for acceptance tests")
+			}
+
+			client := retryablehttp.NewClient()
+
+			service := newUpCloudServiceConnection(username, password, client.HTTPClient)
+
+			objectStorages, err := service.GetObjectStorages()
+			if err != nil {
+				return err
+			}
+
+			for _, objectStorage := range objectStorages.ObjectStorages {
+				_, found := nameMap[objectStorage.Name]
+				if !found {
+					continue
+				}
+
+				print("sweep is deleting objectstore", objectStorage.Name)
```

Comment:
typo `objectstore` => `object storage`

could also use `log` / add line breaks. Though not sure if necessary to log here